### ISSUE TITLE
ViewContainerRef.createComponent with svg component

### ISSUE THREAD

**Issue by leon** [angular/angular#34866]
# 🐞 bug report

### Affected Package
@angular/core

### Is this a regression?
No

### Description

Say I have this svg path component
```ts
@Component({
  selector: 'svg:g[app-path]', // have tried with both svg:g[app-path] and g[app-path]
  templateUrl: './path.component.html',
  styleUrls: ['./path.component.css']
})
export class PathComponent {}
```

When using ViewContainerRef and ComponentFactoryResolver to dynamically load a component.
It doesn't create the element with the correct namespace.

The component is created, but if I look into the properties panel of chrome devtools I can see that it's not correctly being set as a SVGGElement, but instead a HTMLUnknownElement.

I'm guessing this is because somewhere, someone isn't paying attention to that the component getting loaded could be an svg component.

## 🔬 Minimal Reproduction
https://stackblitz.com/edit/angular-dynamically-load-svg?file=README.md

## 🔥 Exception or Error
HTMLUnknownElement instead of SVGGElement

## 🌍  Your Environment

**Angular Version:**
Angular 9-rc.9 locally
And stackblitz, which is running 8, same problem.

**Comment by crisbeto** [comment]
Is this with or without Ivy enabled?

**Comment by leon** [comment]
In my local project I’m running rc9 with ivy.

But stackblitz isn’t using ivy and angular 8

So the problem seems to be with all versions of angular.

**Comment by AndrewKushnir** [comment]
Hi @leon, thanks for reporting the issue.

I've created a [PR with proposed fix](https://github.com/angular/angular/pull/35136). PR still requires review and the final implementation might change, but I'd like to ask if you could test your app with the changes from that PR to make sure the fix covers the case that you have in the app and everything works as expected. You can follow an instruction in [this document on how to use PR artifacts](https://github.com/angular/angular/blob/master/docs/DEVELOPER.md#archives-for-each-package). An archive with updated `@angular/core` package that contains changes from the PR is available [here](https://612284-24195339-gh.circle-artifacts.com/0/angular/core-pr35136-8419aba29d.tgz) (please make sure you update other `@angular` packages to `9.0.0-rc.13` to avoid version conflicts). **Note**: the mentioned archive with updated `@angular/core` package is for **testing purposes only** and not suitable for production usage (that archive will be removed once PR is merged to master).

Thank you.

**Comment by baltie** [comment]
This has been an ongoing problem for quite a while and several issues has been posted. Until this has been fixed, merged and released, this workaround https://github.com/angular/angular/issues/10404#issuecomment-313825690 works nicely enough, at least in latest Angular 8 without Ivy.

**Comment by AndrewKushnir** [comment]
Hi, the changes from the mention PR were included into `9.0.1` release of Angular. Feel free to reopen the ticket if the problem still exists. Thank you.

**Comment by pkozlowski-opensource** [comment]
Duplicate of #10404
